nixos/modules/programs/wayland/sway.nix

```diff
@@ -147,9 +147,8 @@ in
       # https://github.com/emersion/xdg-desktop-portal-wlr/blob/master/contrib/wlroots-portals.conf
       # https://github.com/emersion/xdg-desktop-portal-wlr/pull/315
       xdg.portal.config.sway = {
-        # Use xdg-desktop-portal-gtk for every portal interface...
-        default = "gtk";
-        # ... except for the ScreenCast, Screenshot and Secret
+        default = lib.mkDefault [ "wlr" "gtk" ];
```

Comment:
Do we want to use mkDefault here? If people specify a value, then the list of wlr and gtk is dropped instead of added.

Reply:
```suggestion
        default = [ "wlr" "gtk" ];
```

Reply:
I keep forgetting that string and lists behave differently. And I kind of agree.